.ci/lib.sh

```diff
@@ -85,8 +85,15 @@ function build() {
 function build_and_install() {
 	github_project="$1"
 	make_target="$2"
+	test_not_gopath_set="$3"
+
 	build "${github_project}" "${make_target}"
 	pushd "${GOPATH}/src/${github_project}"
+	if [ "$test_not_gopath_set" = "true" ]; then
+		info "Installing ${github_project} in GOPATH not set mode"
+		sudo -E KATA_RUNTIME="${KATA_RUNTIME}" make install
```

Comment:
`GOPATH` is set on line 89 and you're using `sudo -E` so isn't `GOPATH` set here?

Reply:
I think it found the issue of "GOPATH not set mode" with this test.
I need to fix it in runtime first.
https://github.com/kata-containers/runtime/pull/1286

Reply:
I still don't understand this - `GOPATH` *is* set here so how can it be *"No GO command or GOPATH not set mode"* ?

Reply:
@jodh-intel This part use:
```
sudo -E KATA_RUNTIME="${KATA_RUNTIME}" make install
```
without 
```
PATH="$PATH"
```
Then no go command can work.

Reply:
Doh! I always read `-E` as "export the entire env" rather than "export the vars you give me" :smile: